src/uct/api/uct.h

```diff
@@ -434,6 +434,11 @@ typedef enum uct_atomic_op {
 #define UCT_IFACE_FLAG_TAG_EAGER_BCOPY UCS_BIT(51) /**< Hardware tag matching bcopy eager support */
 #define UCT_IFACE_FLAG_TAG_EAGER_ZCOPY UCS_BIT(52) /**< Hardware tag matching zcopy eager support */
 #define UCT_IFACE_FLAG_TAG_RNDV_ZCOPY  UCS_BIT(53) /**< Hardware tag matching rendezvous zcopy support */
+
+        /* Miscellaneous */
+#define UCT_IFACE_FLAG_TX_RX_DEPENDENCY UCS_BIT(54) /**< TX operations depend on unrelated RX operations
```

Comment:
since this is a performance-related property, IMO we should add it to uct_iface_estimate_perf rather than uct_iface_query

Reply:
fixed, added `->flags` member to counter parameter count inflation.